src/Extension/MemberExtension.php

```diff
@@ -92,6 +121,18 @@ public function currentUserCanViewMFAConfig()
      * @return bool
      */
     public function currentUserCanEditMFAConfig()
+    {
+        return $this->currentUserIsThisMember();
+    }
+
+    /**
+     * Determines whether the logged in user is the member record this extension is applied to.
+     * @see currentUserCanEditMFAConfig
+     * @see updateCMSFields
+     *
+     * @return bool
+     */
+    public function currentUserIsThisMember()
```

Comment:
I'm not sure I really agree with adding this method. Personally I think it's bad practise to have so much logic applied to models (even if that's through extensions) but in this case I think it's really arbitrary. The existing implementation was at least quite specific to MFA. Can you just move the logic to the FormField itself?